src/convenience/keyboard.ts

```diff
@@ -1041,13 +1027,17 @@ function transpose<T>(grid: T[][]): T[][] {
     }
     return transposed;
 }
+interface WrapOptions {
+    /** Set to `true` to completely fill up the last row */
+    fillLastRow?: boolean;
```

Comment:
I think i liked the third argument better than another object. You could just see the default value, its way more speaking to use options rather than a bool and I still need to read the documentation (or try it) as `fillLastRow` or even the comment here is just not that helpful in my opinion.

Having the object there is helpful for future improvements so its not a breaking change to add something while keeping it easy to use but still…

Reply:
What's your suggestion for naming it then? I am not willing to drop this feature for reasons of completeness, but I also can't address the feedback of you guys if you dislike all suggestions without any constructive criticism 🫠
The idea is very easy: make sure to fill up the bottom row with buttons, rather than the top row. Now, how to call it? :)